### PR TITLE
Feat: Adds Global tokens doc pages

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

Global Tokens

- [x] Colors
- [x] Grid & Layout
- [x] Interactive controls
- [x] Refinements
- [x] Spacing
- [x] Typography : missing [Using a custom font section](https://www.faststore.dev/how-to-guides/theming/global-tokens/typography#using-a-custom-font
)

### CODE REVIEW

**Review by renatamottam — APPROVED** [review]
Niiice!

File: apps/site/components/Tokens/TokenColors.module.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.tokenColors {
+  /* display: grid;
+  grid-auto-flow: column dense; */
```
Comment (renatamottam) [inline apps/site/components/Tokens/TokenColors.module.css:3]: Maybe remove this?
```suggestion
```

File: apps/site/pages/docs/global-tokens/Colors.mdx [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+  TokenColors,
+  TokenColor,
+  TokenTable,
+  TokenRow,
+  TokenDivider,
+} from 'site/components/Tokens'
+
+<header>
+
+# Colors
```
Comment (renatamottam) [inline apps/site/pages/docs/global-tokens/Colors.mdx:11]: Shouldn't this file be named as `colors.mdx`? (all small caps)

File: apps/site/components/Tokens/TokenColors.module.css (hunk 1 above)
Comment (hellofanny) [reply apps/site/components/Tokens/TokenColors.module.css:3]: hahah yes! My bad!

File: apps/site/pages/docs/global-tokens/Colors.mdx (hunk 2 above)
Comment (hellofanny) [reply apps/site/pages/docs/global-tokens/Colors.mdx:11]: oh, sure. I thought I've done that. 🙈

File: apps/site/components/Tokens/TokenColor.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import styles from './TokenColors.module.css'
```
Comment (eduardoformiga) [inline apps/site/components/Tokens/TokenColor.tsx:1]: what about using kebab-case?

File: apps/site/pages/docs/global-tokens/spacing.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import { TokenTable, TokenRow, TokenDivider } from 'site/components/Tokens'
```
Comment (eduardoformiga) [inline apps/site/pages/docs/global-tokens/spacing.mdx:1]:
```suggestion
import { TokenTable, TokenRow } from 'site/components/Tokens'
```

**Review by eduardoformiga — APPROVED** [review]
LGTM!

File: apps/site/components/Tokens/TokenColor.tsx (hunk 3 above)
Comment (hellofanny) [reply apps/site/components/Tokens/TokenColor.tsx:1]: ok! I will update all the files then :)

**Pull request merged**